plugin/helpers.go

```diff
@@ -29,8 +33,18 @@ type Helpers interface {
 	//
 	// Minimum server version: 5.6
 	KVSetWithExpiryJSON(key string, value interface{}, expireInSeconds int64) error
+
+	// Register a command, using the Plugin API, and stores the callback attached to the command trigger.
+	// Returns error if the callback is not provided or the API.RegisterCommand fails
+	RegisterCommand(command *model.Command, callback CommandCallback) error
+
+	// Execute a command and executes the stored callback for that trigger.
```

Comment:
Similarly, let's replace all three lines and do:
```suggestion
	// ExecuteCommand handles command execution, triggering previously registered callbacks. 
	// It is meant to replace the implementation of the hook of the same name:
	//     func (p *Plugin) ExecuteCommand(c *Context, args *model.CommandArgs) {
	//         return p.Helpers.ExecuteCommand(c, args)
	//     }
```